tests/test_health_operations.py

```diff
@@ -0,0 +1,143 @@
+import json
+import logging
+import pytest
+
+from utils.utils import get_noobaa_health_status
+import noobaa_sa.exceptions as e
+
+log = logging.getLogger(__name__)
+
+
+class Test_health_operations:
+    def setup_prereqs(self, unique_resource_name, random_hex, account_manager, bucket_manager):
+        """
+            Create account and bucket for performing health operation
+        """
+        account_name = unique_resource_name(prefix="account")
+        access_key = random_hex()
+        secret_key = random_hex()
+        account_manager.create(account_name, access_key, secret_key)
+        account_manager.list()
+        bucket_name = unique_resource_name(prefix="bucket")
+        bucket_manager.create(account_name, bucket_name)
+
+    # Noobaa port health operation
+    @pytest.mark.parametrize(
+        argnames="flag",
+        argvalues=[
+            pytest.param(
+                    {
+                        "https_port": 6443
+                    },
+            ),
+            pytest.param(
+                    {
+                        "https_port": None
+                    },
+            )
+        ],
```

Comment:
In all three test functions you can just pass the value instead of dictionaries as pytest params, and then either construct the dictionary in the test itself, or if it works (I'm not a 100% sure if it does), pass it directly to `get_noobaa_health_status`:
```
    @pytest.mark.parametrize(
        argnames="https_port_param",
        argvalues=[
            pytest.param(True),
            pytest.param(False),
            pytest.param(None),
        ],
        ids=[
             "true_value",
             "false_value",
             "default_value",
        ],
    )
    def test_noobaa_account_flag(self, unique_resource_name, random_hex, account_manager, bucket_manager, https_port_param):
    ...
    ...
    ...
    get_info = json.loads(get_noobaa_health_status(https_port=https_port_param))
```
If this works, then I'm pretty sure there's also a more elegant way to pass the pytest params than this example above.